docs/design/AnoncredsW3CCompatibility.md

```diff
@@ -0,0 +1,286 @@
+# Anoncreds W3C Compatibility
+
+This design proposes to extend the Aries Cloud Agent Python (ACA-PY) to support Hyperledger AnonCreds credentials and presentations in the W3C Verifiable Credentials (VC) and Verifiable Presentations (VP) Format. The aim is to transition from the legacy AnonCreds format specified in Aries-Legacy-Method to the W3C VC format.
+<br><br>
+
+## Overview
+
+We aim to wrap the enhancements made on the Rust Framework for Anoncreds first. Then we will work on the integration of AnonCreds with W3C VC Format in ACA-PY, which includes support for issuing, verifying, and managing W3C VC Format AnonCreds credentials.
+
+Ideally the signatures will be delivered in parallel with the Javascript Framework Document.
+<br><br>
+
+## Caveats
+
+For now, we will only target compatibility with [VCDM (Verifiable Credential Data Model) 1.1](https://www.w3.org/TR/vc-data-model/) because the Rust framework we are deriving from is also working with that target. This means we would support the features being implemented in the RUST frameworks, which include:
```

Comment:
I don't think that ACA-Py "cares" about which version is used -- I think the features will all be the same. From what I understand, the impacts are minimal from the AnonCreds W3C VC/VP formats -- different `@context` files, and the name of the `issuanceDate` field.  I think everything else is the same.

Reply:
There may be some tweaks required to get v1 and v2 of the VC data model working correctly. Some checks are done on the issuanceDate fields I believe, as well as on the exact values of the `@context` (MUST contain the `v1` context url).

Reply:
Given that ACA-Py wants to treat versions of VCDM as minor variations rather than radically different things, I think the preferred answer to [this question](https://github.com/hyperledger/aries-cloudagent-python/pull/2674#discussion_r1428340520) which @swcurran raises below is probably his first proposal -- a flag passed as input -- instead of a different endpoint.

Reply:
Great! We're gonna settle for flag parameters for now.. thank you for the insights and suggestions!